### ISSUE TITLE
Pointer hover zone too small for terminal/code tile divide

### ISSUE THREAD

**Issue by Dan-Heintzelman** [zed-industries/zed#48214]
### Reproduction steps

1. Open ZED
2. CMD + SHIFT +P > Toggle terminal (to open a new terminal
3. Try to make the terminal bigger  by hovering over the divider line between terminal and code tile.
4. The resize icon comes on for a brief blip at the cursor moves over to hover on it, and disappears.  In order to resize I have to precisely point on the top line (above horizontal scroll bar) and click/drag
5. This started happening maybe like 5 or so releases ago

### Current vs. Expected behavior

Current:
Impossible to hover cursor on dividing line between code tiles and terminal area without it disappearing. It is very difficult to resize

Expected:
There should be a sizeable area wherein a cursor is hovered, the resize icon appears at which point the end user can click(select) + hold + and resize.

### Zed version and system specs

Zed 0.221.5 
03bfbf242c57f7fdb45708d63d74182898edf2c5

0.221.5+stable.141.03bfbf242c57f7fdb45708d63d74182898edf2c5

<img width="392" height="611" alt="Image" src="https://github.com/user-attachments/assets/e807e1eb-b5fb-49c0-ad90-116da03b1393" />

### Attach Zed log file

<details><summary>Zed.log</summary>

```log
2026-02-02T14:31:48-06:00 ERROR [gpui::app] timed out waiting on app_will_quit
2026-02-02T14:31:55-06:00 INFO  [zed] ========== starting zed version 0.220.3+stable.127.3d02817699175909ee72bf28305997094c5cef9d, sha 3d02817 ==========
2026-02-02T14:31:55-06:00 INFO  [crashes] spawning crash handler process
2026-02-02T14:31:55-06:00 INFO  [zed] Using git binary path: "/Applications/Zed.app/Contents/MacOS/git"
2026-02-02T14:31:55-06:00 INFO  [crashes] connected to crash handler process after 100ms
2026-02-02T14:31:55-06:00 INFO  [crashes] crash handler registered
2026-02-02T14:31:55-06:00 INFO  [extension_host] extensions updated. loading 23, reloading 0, unloading 0
2026-02-02T14:31:55-06:00 INFO  [auto_update] Auto Update: checking for updates
2026-02-02T14:31:55-06:00 INFO  [project::trusted_worktrees] Worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" is not trusted
2026-02-02T14:31:55-06:00 INFO  [workspace] Rendered first frame
2026-02-02T14:31:55-06:00 INFO  [client] set status on client 0: Authenticating
2026-02-02T14:31:55-06:00 INFO  [project::lsp_store] Waiting for worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" to be trusted, before starting language server yaml-language-server
2026-02-02T14:31:55-06:00 INFO  [extension_host] installing extension catppuccin 0.2.25
2026-02-02T14:31:55-06:00 WARN  [supermaven] unhandled message: Metadata(SupermavenMetadataMessage { dust_strings: None })
2026-02-02T14:31:55-06:00 INFO  [client] set status on client 146044: Authenticated
2026-02-02T14:31:56-06:00 WARN  [supermaven] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2026-02-02T14:31:56-06:00 INFO  [extension_host] installing extension ruby 0.16.5
2026-02-02T14:31:56-06:00 INFO  [util] set environment variables from shell:/bin/zsh, path:/Users/danheintzelman/.asdf/shims:/opt/homebrew/opt/asdf/libexec/bin:/Users/danheintzelman/.nvm/versions/node/v21.1.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin
2026-02-02T14:31:56-06:00 INFO  [node_runtime] Node runtime install_if_needed
2026-02-02T14:31:56-06:00 INFO  [project::lsp_store] Waiting for worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" to be trusted, before starting language server solargraph
2026-02-02T14:31:56-06:00 INFO  [extension_host] rebuilt extension index in 29.7775ms
2026-02-02T14:31:56-06:00 INFO  [extension_host] extensions updated. loading 0, reloading 1, unloading 1
2026-02-02T14:31:56-06:00 WARN  [node_runtime] using Zed managed Node.js at /Users/danheintzelman/Library/Application Support/Zed/node/node-v24.11.0-darwin-arm64 since checking system Node.js failed with error: node at /Users/danheintzelman/.nvm/versions/node/v21.1.0/bin/node is too old. want: 22.0.0, got: 21.1.0
2026-02-02T14:31:58-06:00 INFO  [extension_host] rebuilt extension index in 20.984375ms
2026-02-02T14:31:58-06:00 INFO  [extension_host] extensions updated. loading 1, reloading 0, unloading 0
2026-02-02T14:31:58-06:00 INFO  [extension_host] installing extension ultimate-dark-neo 0.1.0
2026-02-02T14:31:58-06:00 INFO  [extension_host] rebuilt extension index in 23.446958ms
2026-02-02T14:31:58-06:00 INFO  [extension_host] extensions updated. loading 0, reloading 1, unloading 0
2026-02-02T14:32:01-06:00 INFO  [auto_update] downloaded update. path:"/var/folders/4c/821gz4hj2d98msyp0lnlm3900000gn/T/zed-auto-updateDeaTx2/Zed.dmg"
2026-02-02T14:32:06-06:00 INFO  [auto_update] Successfully unmounted the disk image
2026-02-02T14:32:08-06:00 ERROR [gpui::app] timed out waiting on app_will_quit
2026-02-02T14:32:10-06:00 INFO  [zed] ========== starting zed version 0.221.5+stable.141.03bfbf242c57f7fdb45708d63d74182898edf2c5, sha 03bfbf2 ==========
2026-02-02T14:32:10-06:00 INFO  [crashes] spawning crash handler process
2026-02-02T14:32:10-06:00 INFO  [zed] Using git binary path: "/Applications/Zed.app/Contents/MacOS/git"
2026-02-02T14:32:10-06:00 INFO  [crashes] connected to crash handler process after 100ms
2026-02-02T14:32:10-06:00 INFO  [crashes] crash handler registered
2026-02-02T14:32:10-06:00 INFO  [sqlez::migrations] Found 17 foreign key relationships to check
2026-02-02T14:32:10-06:00 INFO  [extension_host] extensions updated. loading 23, reloading 0, unloading 0
2026-02-02T14:32:10-06:00 INFO  [auto_update] Auto Update: checking for updates
2026-02-02T14:32:10-06:00 INFO  [project::trusted_worktrees] Worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" is not trusted
2026-02-02T14:32:10-06:00 INFO  [workspace] Rendered first frame
2026-02-02T14:32:10-06:00 INFO  [client] set status on client 0: Authenticating
2026-02-02T14:32:10-06:00 INFO  [project::lsp_store] Waiting for worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" to be trusted, before starting language server yaml-language-server
2026-02-02T14:32:11-06:00 WARN  [supermaven] unhandled message: Metadata(SupermavenMetadataMessage { dust_strings: None })
2026-02-02T14:32:11-06:00 INFO  [client] set status on client 146044: Authenticated
2026-02-02T14:32:11-06:00 INFO  [util] set environment variables from shell:/bin/zsh, path:/Users/danheintzelman/.asdf/shims:/opt/homebrew/opt/asdf/libexec/bin:/Users/danheintzelman/.nvm/versions/node/v21.1.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin:/Users/danheintzelman/.nvm/versions/node/v21.1.0/bin
2026-02-02T14:32:11-06:00 INFO  [project::lsp_store] Waiting for worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" to be trusted, before starting language server solargraph
2026-02-02T14:32:11-06:00 WARN  [supermaven] unhandled message: Set(SupermavenSetMessage { key: "disabled", value: Bool(false) })
2026-02-02T14:32:11-06:00 INFO  [node_runtime] Node runtime install_if_needed
2026-02-02T14:32:11-06:00 WARN  [node_runtime] using Zed managed Node.js at /Users/danheintzelman/Library/Application Support/Zed/node/node-v24.11.0-darwin-arm64 since checking system Node.js failed with error: node at /Users/danheintzelman/.nvm/versions/node/v21.1.0/bin/node is too old. want: 22.0.0, got: 21.1.0
2026-02-02T14:32:26-06:00 INFO  [auto_update] Auto Update: checking for updates
2026-02-02T14:33:09-06:00 INFO  [project::trusted_worktrees] Worktree "/Users/danheintzelman/dev/repos/prevail" is not trusted
2026-02-02T14:33:09-06:00 INFO  [project::lsp_store] Worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" is trusted, starting language server yaml-language-server
2026-02-02T14:33:09-06:00 INFO  [project::lsp_store] Worktree "/Users/danheintzelman/dev/repos/heartbeat-testing" is trusted, starting language server solargraph
2026-02-02T14:33:09-06:00 INFO  [project::lsp_store] Waiting for worktree "/Users/danheintzelman/dev/repos/prevail" to be trusted, before starting language server solargraph
2026-02-02T14:33:09-06:00 WARN  [language] unrecognized capture name 'indent.begin' in RBS indents TreeSitter query (suppress this warning by prefixing with '_')
2026-02-02T14:33:09-06:00 WARN  [language] unrecognized capture name 'indent.ignore' in RBS indents TreeSitter query (suppress this warning by prefixing with '_')
2026-02-02T14:33:09-06:00 ERROR [crates/workspace/src/persistence/model.rs:303] Failed to open path in project

Caused by:
    Could not find entry in worktree for "Gemfile.lock" after refresh
2026-02-02T14:33:09-06:00 ERROR [crates/workspace/src/persistence/model.rs:303] Failed to open path in project

Caused by:
    Could not find entry in worktree for "app/models/organization.rb" after refresh
2026-02-02T14:33:09-06:00 ERROR [crates/workspace/src/persistence/model.rs:303] Failed to open path in project

Caused by:
    Could not find entry in worktree for "app/services/import_service/base_import_service.rb" after refresh
2026-02-02T14:33:10-06:00 INFO  [project::prettier_store] Initializing default prettier with plugins {"@prettier/plugin-xml"}
2026-02-02T14:33:10-06:00 INFO  [project::prettier_store] Installing default prettier and plugins: [("@prettier/plugin-xml", "3.4.2"), ("prettier", "3.8.1")]
2026-02-02T14:33:11-06:00 INFO  [project::prettier_store] Initialized default prettier with plugins: {"@prettier/plugin-xml"}
2026-02-02T14:36:40-06:00 ERROR [workspace::notifications] Showing error notification in app: Error: dev::ToggleInspector is only available in debug builds

```

</details>

### Relevant Zed settings

<details><summary>settings.json</summary>

```json

{
  "hard_tabs": false,
  "tab_size": 2,
  "lsp": {
    "steep": {
      "settings": {
        "require_root_steepfile": false
      }
    },
    "sorbet": {
      "initialization_options": {
        "enabled": false
      }
    }
  },
  "languages": {
    "Ruby": {
      "language_servers": ["ruby-lsp", "!sorbet", "!steep", "..."]
    }
  }
}
```

</details>

### Relevant Keymap

<details><summary>keymap.json</summary>

```json
no keymap
```

</details>

### If you are using WSL on Windows, what flavor of Linux are you using?

None
